pipeline/scripts/ci/getPipelineStages.py

```diff
@@ -112,14 +119,21 @@ def fetch_stages(args):
             os.makedirs(logdir)
             execute_cli += f" --log-dir {logdir}"
             cleanup_cli += f" --cloud {cloud_type}"
+            script.update(ibmc_overrides)
+        else:
+            script.update(openstack_overrides)
```

Comment:
I think these lines can be removed. We should change the ln:126 to

```
if metadata_overrides and cloud_type in metadata_overrides.keys():
  script.update(metadata_overrides[cloud_type]
```

Reply:
I added this logic because I wanted to pop out the cloud_type specific overrides from the metadata_overrides dictionary, because in the next line I am doing `script.update(metadata_overrides)` so I wouldn't want metadata_overrides to have cloud type specific overrides at this point.